client-v2/src/shared/components/snapLink/SnapLink.tsx

```diff
@@ -121,7 +128,21 @@ const SnapLink = ({
             </SnapLinkURL>
           </CollectionItemHeadingContainer>
         </CollectionItemContent>
-        {size === 'default' && <ThumbnailSmall />}
+        <ImageWrapper>
+          <ThumbnailSmall
+            imageHide={article && article.imageHide}
+            url={article && article.thumbnail}
+          >
+            {article && article.thumbnail ? (
+              <ThumbnailCutout src={article && article.thumbnail} />
```

Comment:
I don't think it's possible for a snap link to have a default cutout image -- these come from the `cutoutThumbnail` property, which is derived by `getContributorImage`.  So `ThumbnailCutout` here should be unnecessary.